projects/igniteui-angular/src/lib/grids/grid/grid.sorting.spec.ts

```diff
@@ -1,27 +1,35 @@
-import { Component, ViewChild } from '@angular/core';
 import { async, TestBed, fakeAsync } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
 import { SortingDirection } from '../../data-operations/sorting-expression.interface';
 import { IgxGridComponent } from './grid.component';
 import { IgxGridModule } from './index';
-import { GridTemplateStrings, ColumnDefinitions } from '../../test-utils/template-strings.spec';
-import { BasicGridComponent } from '../../test-utils/grid-base-components.spec';
-import { SampleTestData } from '../../test-utils/sample-test-data.spec';
-import { DefaultSortingStrategy, ISortingStrategy, NoopSortingStrategy } from '../../data-operations/sorting-strategy';
+import { DefaultSortingStrategy, NoopSortingStrategy } from '../../data-operations/sorting-strategy';
 import { IgxGridCellComponent } from '../cell.component';
 import { configureTestSuite } from '../../test-utils/configure-suite';
 import { IgxGridFilteringRowComponent } from '../filtering/base/grid-filtering-row.component';
 import { IgxChipComponent } from '../../chips/chip.component';
 import { NoopAnimationsModule } from '@angular/platform-browser/animations';
 import { GridFunctions } from '../../test-utils/grid-functions.spec';
+import { GridDeclaredColumnsComponent, SortByParityComponent } from '../../test-utils/grid-samples.spec';
+import { ControlsFunction } from '../../test-utils/controls-functions.spec';
+import { UIInteractions } from '../../test-utils/ui-interactions.spec';
 
 const SORTING_ICON_ASC_CONTENT = 'arrow_upward';
 const SORTING_ICON_DESC_CONTENT = 'arrow_downward';
 
+
+function getCurrentCellFromGrid(grid, rowIndex, cellIndex) {
```

Comment:
Can you use `getCurrentCellFromGrid` that is imported from GridFunctions instead, we do not need to define another function with the same logic.

Example:
Change this:
`expect(GridFunctions.getValueFromCellElement(getCurrentCellFromGrid(grid, 0, 1))).toEqual('Jane');`
With:
`expect(GridFunctions.getValueFromCellElement(GridFunctions.getCurrentCellFromGrid(grid, 0, 1))).toEqual('Jane');`